content/src/view.js

```diff
@@ -2908,6 +2908,7 @@ function noteNewFilename(pane, filename) {
     paneState.dropletEditor.setMode(
         dropletModeForMimeType(visibleMimeType),
         dropletOptionsForMimeType(visibleMimeType));
+    paneState.dropletEditor.setPalette(paletteForPane(paneState));
```

Comment:
Let me know if you've verified this works correctly when there is no block support (e.g., .txt or .xml files)?